data/huion-kd100.tablet

```diff
@@ -0,0 +1,56 @@
+# Huion
+# KD100
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+# 
+#   S
+#
+# *---------*
+# |         |
+# | A B C D |
+# |         |
+# | E F G H |
+# |         |
+# | I J K L |
+# |         |
+# | M N O P |
+# |       P |
+# | QQQ R P |
+# |         |
+# *---------*
+#
+#
+# Ring:
+# (a=strip min, A=strip max)
+#
+#   L R
+#
+# *---------*
+# |         |
+# |         |
+# |         |
+# *---------*
+#
+
+[Device]
+Name=Huion KD100
+DeviceMatch=usb:256c:006d:HUION Huion Tablet_KD100 Keyboard;usb:256c:006d:HUION Huion Tablet_KD100 Pad;usb:256c:006d:HUION Huion Tablet_KD100 Consumer;usb:256c:006d:HUION Huion Tablet_KD100 Pen;usb:256c:006d:HUION Huion Tablet_KD100
```

Comment:
this isn't the right match, you don't want libwacom to match on all event nodes this device produces, only on the one that sends the actual events.

Reply:
Got it now